tests/src/JIT/Regression/JitBlue/GitHub_5047/project.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
Can this test rely on one of the stock project.json?  There are other tests that don't need this.  @pgavlin I thought we were not using individual project.json files in the test tree.